shared/protocol/src/main/java/io/pravega/shared/protocol/netty/WireCommands.java

```diff
@@ -2182,6 +2189,78 @@ public boolean isFailure() {
         }
     }
 
+    @Data
+    public static final class ReadTableEntriesDelta implements Request, WireCommand {
+        final WireCommandType type = WireCommandType.READ_TABLE_ENTRIES_DELTA;
+        final long requestId;
+        final String segment;
+        @ToString.Exclude
+        final String delegationToken;
+        final long fromVersion;
+        final int suggestedEntryCount;
+
+        @Override
+        public void process(RequestProcessor cp) {
+
+        }
+
+        @Override
+        public void writeFields(DataOutput out) throws IOException {
+            out.writeLong(requestId);
+            out.writeUTF(segment);
+            out.writeUTF(delegationToken == null ? "" : delegationToken);
+            out.writeLong(fromVersion);
+            out.writeInt(suggestedEntryCount);
+        }
+
+        public static WireCommand readFrom(ByteBufInputStream in, int length) throws IOException {
+            long requestId = in.readLong();
+            String segment = in.readUTF();
+            String delegationToken = in.readUTF();
+            long fromVersion = in.readLong();
+            int suggestedEntryCount = in.readInt();
+
+            return new ReadTableEntriesDelta(requestId, segment, delegationToken, fromVersion, suggestedEntryCount);
+        }
+    }
+
+    @Data
+    public static final class TableEntriesDeltaRead implements Reply, WireCommand {
+        final WireCommandType type = WireCommandType.TABLE_ENTRIES_DELTA_READ;
+        final long requestId;
+        final String segment;
+        final TableEntries tableEntries;
+        final boolean shouldClear;
+        final boolean reachedEnd;
+        final long lastVersion;
+
+        @Override
+        public void process(ReplyProcessor cp) {
+
```

Comment:
Have this throw `UnsupportedOperationException` and add a comment that it will be implemented with 
https://github.com/pravega/pravega/issues/4677